src/efisecdb.c

```diff
@@ -186,13 +186,20 @@ add_action(list_t *list, action_type_t action_type, const efi_guid_t *owner,
 	list_add_tail(&action->list, list);
 }
 
+/*
+ * These need to be static globals so that they're reachable from atexit handler.
```

Comment:
line length

Reply:
i removed the comment. I don't think its needed.